src/lib/abacus/websocket.ts

```diff
@@ -144,6 +145,23 @@ class AbacusWebsocket implements Omit<AbacusWebsocketProtocol, '__doNotUseOrImpl
 
               break;
             }
+            case 'v4_markets': {
+              if (testFlags.displayInitializingMarkets) {
+                shouldProcess = false;
+                const { contents } = parsedMessage;
+
+                Object.keys(contents.markets)?.forEach((market: any) => {
```

Comment:
do we need the `?.`, or if `contents.markets` can be undefined, should we do `Object.keys(contents.markets ?? {})`?